packages/react-native-platform-override/src/Cli.ts

```diff
@@ -382,36 +396,46 @@ async function checkFileExists(friendlyName: string, filePath: string) {
 /**
  * Read a manifest and print a pretty error if we can't
  */
-async function readManifest(file: string, version?: string): Promise<Manifest> {
-  const gitReactRepo = await GitReactFileRepository.createAndInit();
-  const ovrRepo = new OverrideFileRepositoryImpl(path.dirname(file));
+async function readManifest(file: string): Promise<Manifest> {
+  try {
+    const data = await Serialized.readManifestFromFile(file);
+    return Manifest.fromSerialized(data);
+  } catch (ex) {
+    throw new Error('Could not parse manifest. Is it valid?');
```

Comment:
I should throw this lower so we can get more descriptive messages when this isn't the case.